src/main.js

```diff
@@ -54,6 +69,7 @@ module.exports = {
       idleCallbacks.delete(callbackID)
       if (!atom.inSpecMode()) {
         require('atom-package-deps').install('linter-eslint')
```

Comment:
Very much not. That takes ~50 ms to run, and only needs to be done once. `loadDeps` needs to be checked every time it is about to use a dependency.

Reply:
All the other dependencies only need to be loaded once also. Load deps is essentially a no-op after the first time it's called. It will be wrapped in an if block just like all the others.

Reply:
We can also cache the call to `atom.inSpecMode()` and `if` against the variable instead of the function.

Reply:
That would require adding a flag to check whether it had been called or not. When it isn't a code dependency I really don't think it belongs in there.